### PR TITLE
order status feature show karna baaki hai -ak

### PR DESCRIPTION
**Pull request by ayushkedia1990**

### CODE REVIEW

File: src/orderconfirmpage/OrderConfirmPage.jsx [hunk 1]
```diff
@@ -12,12 +12,17 @@ import React, { useEffect } from "react";
...
 import { connect } from "react-redux";
 import { createStructuredSelector } from "reselect";
-import s from './OrderConfirmPage.css'
+import s from './OrderConfirmPage.css';
+import { useHistory } from "react-router-dom/cjs/react-router-dom.min";
+
 
 import { selectCartTotal } from "../redux/cart/cart.selectors";
+import PaymentStatus from "./paymentStatus";
+import { Redirect } from "react-router-dom/cjs/react-router-dom.min";
```
Comment (arijeet-mukherjee) [inline src/orderconfirmpage/OrderConfirmPage.jsx:21]: We can remove this

Comment (cypherpoll-0) [reply src/orderconfirmpage/OrderConfirmPage.jsx:21]: fuck off

**Review by arijeet-mukherjee — APPROVED** [review]
We will remove this redirect in next PR

**Comment by ayushkedia1990** [comment]
> We will remove this redirect in next PR

accha theek hai dekhte hain abhi ke good night shabha khair

**Pull request merged**